src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
This does not make it a breraking change.  Removing the fields will make it a breaking change.